CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,219 @@
+
+**Response and Potential Consequences**
+
+The Quansight COC Committee will recommend a response to the Quansight Labs Co-Directors (or higher up the Quansight leadership team if appropriate).
+
+Potential consequences (for reportees) of Quansight Code of Conduct incidents include:
+
+* Nothing (if it is determined that no breach occurred)
+* Private feedback or reprimand from Quansight to the reportee(s) involved
+* Warning the reportee to cease their behavior and that any further incident reports will result in sanctions
+* A public announcement that an incident occurred
+* Mediation (only if both reporter(s) and reportee(s) agree)
+* A permanent or temporary ban from some or all Quansight spaces (mailing lists, GitHub repos, in-person events, etc.)
+* Assistance to the complainant with a report to other bodies (e.g., institutional offices or appropriate law enforcement agencies)
+* An imposed vacation or removal from Quansight employment/affiliation (for Quansight personnel/affiliates)
+* Publishing an account of the incident and calling for the resignation of the reportee(s) from their responsibilities (usually pursued when Quansight lacks formal authority: this may be appropriate if the person is an external event leader, if they refuse to recuse themselves from a conflict of interest, or similar)
+* Any other response that Quansight deems necessary and appropriate to the situation
+* At Quansight events, if a participant engages in behavior that violates this Code of Conduct, the organizers and staff may take any action they deem appropriate
+
+At an in-person Quansight-organized event, potential consequences for Code of Conduct incidents include:
+
+* Warning the offender to cease their behavior and that any further incidents will result in sanctions
+* Requiring that the offender avoid any interaction with—and physical proximity to—any relevant person(s) (e.g., in the case of harrassment) for the remainder of the event
+* Immediately ending all the offender's remaining event responsibilities and privileges
+* Expelling the offender from the event immediately and requiring that they do not return
+* Banning the person from future events (either indefinitely or for a certain time period)
+* Any other response that Quansight deems necessary and appropriate to the situation
+
+No one espousing views or values contrary to the standards of our Code of Conduct will be permitted to hold any position representing Quansight (volunteer positions included). Quansight has the right and responsibility to remove, edit, or reject comments, commits, code, edits, and other online contributions that are not aligned with this Code of Conduct.
+
+## 3. License
```

Comment:
This isn't actually a license, strictly speaking.

Reply:
It seems most the CoCs listed below use a CC-BY license. The Ubuntu one uses a CC-BY-SA license, and SciPy and NeurIPS don't list a license (but SciPy is presumably under the BSD license of the rest of SciPy).

Reply:
Okay, yes, this isn't a License... and we don't have one in the directory either.

I'm going to punt the question of choosing a license for now (unless someone has a strong preference for a particular license and they want to add in the relevant LICENSE document.

Reply:
> but SciPy is presumably under the BSD license of the rest of SciPy

yep. I don't believe in licensing "other content" separately from "docs" and that again separately from "code".

> unless someone has a strong preference for a particular license and they want to add in the relevant LICENSE document

please don't add in a full-fledged license. just one line and a link.